game/gamestates/settings.lua

```diff
@@ -48,8 +45,8 @@ function state:init()
   _selection = 1
 end
 
-function state:enter(from, soundtrack)
-  _soundtrack = soundtrack
+function state:enter(from)
+  _soundtrack = Util.findId("soundtrack")
```

Comment:
If it's going to be globally accessible, why not just make it a singleton?

Reply:
Done